enable/__init__.py

```diff
@@ -20,6 +20,6 @@
     __requires__.append('pillow')
 
 __extras_require__ = {
-    'demo': ['chaco', 'mayavi', 'scipy', 'traitsui[demo]'],
+    'demo': ['chaco', 'mayavi', 'scipy', 'traitsui[demo]', 'kiwisolver'],
```

Comment:
This is a bit orthogonal to this PR, but so many examples depend on `ConstraintsContainer` that it felt appropriate to add.  If one does `pip install enable[demo]` and `pip install etsdemo[pyside2]` and they launch the demo app, they would expect to have what they need to see all the demos.  

I believe there are other dependencies as well (e.g. `pyglet` for `pyglet_gl.py`), but they are much less common.  I may end up adding to them to this PR as well?